src/navs/index.js

```diff
@@ -3988,66 +3988,164 @@ export const docsMenu = {
                         title: 'Beta',
                         className: 'uppercase !bg-blue/10 !text-blue !dark:text-white !dark:bg-blue/50',
                     },
                 },
                 {
                     name: 'Overview',
-                    url: '/docs/ai-engineering',
+                    url: '/docs/llm-analytics',
                     icon: 'IconHome',
                     color: 'seagreen',
                 },
+                {
+                    name: 'Getting started',
+                },
+                {
+                    name: 'Start here',
+                    url: '/docs/llm-analytics/start-here',
+                    icon: 'IconListCheck',
+                    color: 'orange',
+                    featured: true,
+                },
                 {
                     name: 'Installation',
-                    url: '/docs/ai-engineering/observability',
+                    url: '/docs/llm-analytics/installation',
                     icon: 'IconBook',
                     color: 'blue',
                     featured: true,
+                    children: [
+                        {
+                            name: 'Overview',
+                            url: '/docs/llm-analytics/installation',
+                        },
+                        {
+                            name: 'OpenAI',
+                            url: '/docs/llm-analytics/installation/openai',
+                        },
+                        {
+                            name: 'Anthropic',
+                            url: '/docs/llm-analytics/installation/anthropic',
+                        },
+                        {
+                            name: 'Google',
+                            url: '/docs/llm-analytics/installation/google',
+                        },
+                        {
+                            name: 'LangChain',
+                            url: '/docs/llm-analytics/installation/langchain',
+                        },
+                        {
+                            name: 'Vercel AI SDK',
+                            url: '/docs/llm-analytics/installation/vercel-ai',
+                        },
+                    ],
+                },
+                {
+                    name: 'Privacy mode',
+                    url: '/docs/llm-analytics/privacy-mode',
+                    icon: 'IconLock',
+                    color: 'purple',
+                    featured: true,
                 },
                 {
-                    name: 'Dashboard',
-                    url: '/docs/ai-engineering/dashboard',
+                    name: 'Concepts',
+                },
+                {
+                    name: 'Generations',
+                    url: '/docs/llm-analytics/generations',
+                    icon: 'IconAIText',
+                    color: 'seagreen',
+                    featured: true,
+                },
+                {
+                    name: 'Traces',
```

Comment:
I think traces should go below spans as its the collection of spans and generations